### ISSUE TITLE
please let me know what i missed.

### ISSUE THREAD

**Issue by iamadcat** [DRE2N/DungeonsXL#276]
first of all, sorry for my bad skill for english.

1. my first problem is when i tested my dungeon with /dxl test command, all of signs are disappeared and mobs are spawned properly.
but, if i entered the dungeon with /dxl play [name of dungeon], the signs aren't disappears and the mobs aren't also.

2. I made a map and placed signs for mobs and triggers. I can play the dungeon with /dxl test command, but if i play the dungeon with /dxl play, i can't play properly and also those signs aren't disappears. mobs aren't spawns, of course.

In short, I read the wiki so many times, i don't understand about how to making a dungeon for users(not just for test). please give a simple tutorial. (example: should i make a floors, lobbies, signs like lobby or play and so on)

thanks for your great work. you are so professional.

**Comment by Sataniel98** [comment]
Hi,
i think the problem is in your lobby. Test will start the game right away, while play will just teleport you to the lobby. You'll need at least a ready sign sothat most signs get initialized when it is triggered. You can make it available for your users with portals and group signs.

**Comment by iamadcat** [comment]
@Sataniel98 Thanks for your reply. by the way, should i make a lobby or ready sign in the main world? i still don't understand about floor. i just made a dungeon with one map.

**Comment by Sataniel98** [comment]
Just put it somewhere in the dungeon map.
